src/backend/api/reservations.js

```diff
@@ -43,10 +81,25 @@ router.get('/:id', async(req, res) => {
         console.error(err);
         res.status(500).json(err);
     };
+
 });
 
 
 // Updates the reservation by id
+
+router.put('/:id', async (req, res) => {
+  try {
+    const id = req.params.id;
+    await knex('reservation')
+      .where({ id: id })
+      .update(req.body)
+
+    res.json({ "message": "reservation updated" });
```

Comment:
What happens if the `req.body` has some issue and failed to `Update`?
What is your code response?

Reply:
it will respond with console.error(err); and res.status(500).send(err), right?